### DISCUSSION TITLE
Need to clone, basic git command (status, add, commit), and push local repo updates

### DISCUSSION THREAD

**Discussion by 3mp3ri0r** [community/community#12782] [Mobile]
Hi GitHub teams.

I need mobile apps have feature for clone repository, git basic command, and push updates to remote server.

The case that I face is I'm taking notes with apps that use regular .md files as the content (foam and obsidian). I sync my notes from desktop and mobile using git (GitHub especially). I can clone, edit and update remote git server in desktop easily, but it was not the case for mobile.

Please add this feature to remove dependency from another apps. Currently I use MGit in mobile.

Thank you.

Regards,
Christoforus Surjoputro

**Comment by stevepopovich** [comment]
Hi Christoforus, thanks for the feedback. I will pass this along to the team for prioritization!

**Reply by 3mp3ri0r** [reply]
Nice. Thank you @stevepopovich. ❤️

**Reply by ArenMaes** [reply]
@stevepopovich Do you have an update about this? I have the same issue/request.

**Reply by stevepopovich** [reply]
It's not something we are prioritizing at the moment. It is on our list though! Sorry, that's the best news I can give right now.

**Reply by rjvisser** [reply]
@stevepopovich any change since the last response? 

I'm working on the optimization of the workflow in relation to software development projects and would like to be able to handle all documentation in relation to such a project as code (docs as code), thereby increasing the collaboration between the product owners and developers.

Similar to the original request, it would be great to be able to use Git and clone repos in order to access the documents locally on mobile platforms - just like we can on the desktop.

With the markdown files available as local files on the mobile platforms, we can then provide for additional applications to edit these files as well as keep todo's and offer more elaborate project management tasks.

In my opinion the perfect enabler for the most efficient management of software development projects without the need for other expensive tools while assuring the teams of true portability of their code and associates documentation.

Please advise
RJ

**Reply by stevepopovich** [reply]
We have not chosen to implement this at this time, sorry.

**Comment by stevepopovich** [comment]
In the event we choose not to tackle this in the app, I would suggest a service like https://hackmd.io/ that let's you collaborate on markdown files in real-time. Should sync across devices too. Hope that helps!

**Reply by 3mp3ri0r** [reply]
Thanks for the suggestion, @stevepopovich. So far I search hackmd just regular markdown editor, so it does not support wikilink style markdown.
